csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2634,7 +2801,7 @@ private module FlowExploration {
     PartialPathNodePriv mid, PartialAccessPath ap1, Content f, Node node, PartialAccessPath ap2
   ) {
     ap1 = mid.getAp() and
-    store(mid.getNode(), f, node) and
+    storeDirect(mid.getNode(), f, node) and
```

Comment:
You are removing a lot of flow from partial paths.  General getters, setters, and value-flow-through is no longer supported. 
Also, the addition of getter-setters is not supported here. This should all be fixed.